Documentation/UsingSetting/TheConstantEditor.rst

```diff
@@ -6,34 +6,43 @@
 The constant editor
 ===================
 
+..  versionchanged:: 12.0
+    The TypoScript management tools are now found in backend module
+    :guilabel:`Site Management > TypoScript`. It was formerly found in
+    "Page > Template".
+
 It's possible to add comments in TypoScript. Comments are always ignored by the
-parser when the template is processed. But the backend module **WEB > Template**
+parser when the TypoScript is processed. But the backend module
+:guilabel:`Site Management > TypoScript`
 has the ability to use comments in the constant editor to make simple
 configuration of a template even easier than constants already make it
 themselves.
 
-..  include:: /Images/AutomaticScreenshots/TemplatesModul/TemplatesConstantEditor.rst.txt
+..  figure:: /Images/ManualScreenshots/TypoScriptModule/ConstantEditor.png
+    :alt: Screenshot of the TYPO3 Backend showing the constant editor
+
+When the :guilabel:`Constant Editor` parses the TypoScript constants,
+*all* comments before every constant-definition are registered.
 
-When the "Constant Editor" parses the template, *all* comments before every
-constant-definition are registered.  A certain syntax is available to define
+A certain syntax is available to define
```

Comment:
Should we add a note here that explicitly states:

Comments are bound to the constant after a newline. If you use other comments that should NOT apply to a constant (like "#customcategory" [ref]) please insert a distinct newline after it.

(Ah, a similar note comes later; maybe we can add a ref to that note here, and also enhance the later note to explain that the reason for the newline is this "bind to constant" thing?)

Reply:
I would suggest to mention here that the constant needs to be defined in the next line as not to confuse ppl

Reply:
Done